seqr/views/apis/auth_api.py

```diff
@@ -28,7 +29,12 @@ def login_view(request):
     if users.count() != 1:
         return create_json_response({}, status=401, reason='Invalid credentials')
 
-    u = authenticate(username=users.first().username, password=request_json['password'])
+    user = users.first()
+    if SOCIAL_AUTH_GOOGLE_OAUTH2_KEY and (user_is_data_manager(user) or user.is_superuser):
+        logger.warning("Privileged user {} is trying to login without Google authentication.".format(user))
+        return create_json_response({}, status=401, reason='Privileged user must login with Google authentication.')
```

Comment:
so for some reason, the HTTP reason doesn't parse well in the JS in production, so its best to also include a json body with the key `error`. So here that would be 
```
error = 'Privileged user must login with Google authentication.'
return create_json_response({'error': error}, status=401, reason=error)
```